# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/crashtests.list
# Commit: aaa7e1f81404
# Full Hash: aaa7e1f8140455a9f83be8278720598ea3fe4cd8
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2021-12-14 09:42:05
# Description:
#   Bug 1745775 - Fix invalid surface info indexing r=gfx-reviewers,bradwerth
#   
#   Some parts of the visibility pass have been ported to use the
#   picture graph infrastructure (update pass assignment and bounding
#   rect propagation) but the main visibility pass still relies on
# ==============================================================================

diff -r de1608e5f69b -r aaa7e1f81404 gfx/tests/crashtests/crashtests.list
--- a/gfx/tests/crashtests/crashtests.list	Tue Dec 14 00:24:22 2021 +0000
+++ b/gfx/tests/crashtests/crashtests.list	Tue Dec 14 01:06:24 2021 +0000
@@ -208,3 +208,4 @@
 load 1704321-1.html
 load 1702638.html
 load 1730695.html
+load 1745775.html